src/components/buttons/aprove-dev-request.js

```diff
@@ -23,5 +23,18 @@ module.exports = {
     await interaction.message.thread.setArchived(true);
 
     await interaction[interaction.replied || interaction.deferred ? 'followUp' : 'reply']({ content: "Desenvolvedor Aprovado", ephemeral: true });
+  
+    const webhook = parseWebhookURL(process.env.OFFTOPIC_WEBHOOK);
+    await interaction.client.fetchWebhook(webhook.id, webhook.token)
+        .then((webhook) => {
+            webhook.send({
```

Comment:
sempre espera a promise retornar pra se der reject o erro nao ficar perdido na call stack
```suggestion
            await webhook.send({
```